akka-stream/src/main/boilerplate/akka/stream/javadsl/GraphCreate.scala.template

```diff
@@ -22,23 +22,23 @@ private[stream] abstract class GraphCreate {
    */
   def create[S1 <: Shape, S <: Shape, M](g1: Graph[S1, M],
       block: function.Function2[GraphDSL.Builder[M], S1, S]): Graph[S, M] =
-    scaladsl.GraphDSL.create(g1) { b => s => block.apply(b.asJava, s) }
+    scaladsl.GraphDSL.create(g1) { b => s => block.apply(b.asJava, s.asInstanceOf[S1]) }
```

Comment:
This might work if we get the Scala side right, in that case this bail-out-cast should be reverted.